api-docs/interface/server-socket.md

```diff
@@ -38,28 +42,34 @@ Fields:
 - `time` (number): The number of *miliseconds* that are left in the question (where 0 is the end of the question)
 - `score` (number): player’s current score, the host will recieve garbage
 - `username`: the name of the player. 
+future:
+- `questionIndex` : the index of the question started.
 
 ## `endQuestion`
 
 Sent by server when the timer on a question runs out or the host clicks “End now,” and moves clients to the screen to review results
-The same message is sent to the host too except with only the correctAnswers field (maybe a leaderboard later).
+The same message is sent to the host, but the information about the answer choice may be inacurate. 
 
 Fields: 
 - `correctAnswers` (numeric array) : 
 - `score` (number) = the player’s current score, the host will recieve garbage
 - `scoreChange` (number) = how much their score increased due to this question, The host will recieve garbage
-- `correct` (boolean) : if the player's answer to the question is correct. The host will recieve garbagae
-- `leaderboard`: sorted array of [{name: string, score: number }, …]
+- `correct` (boolean) : if the player's answer to the question is correct. The host will recieve garbage
+- `leaderboard`: sorted array of `Leaderboard` objects. 
 - `time` : number: the ammount of time the player took to answer the question
 
+future:
+- `questionIndex` : the index of the question started.
 
 ## `playerAction`
 
-Every time a player completes an action (join, answer) the host is sent this message
-It will contain a player id and username that has completed the most recent action.
+future: 
 
-This may be sent many times rapidly on the case of a host just conneccting. 
+Every time a player completes an action (join, answer) the host is sent this message
+It will contain a player id and username that has completed the most recent action. 
```

Comment:
```suggestion
It will contain a player id and username that has completed the most recent action. It is also sent to the host right after they join with all the players that have done that action.
```